auth/src/test/java/com/firebase/ui/auth/viewmodel/PhoneProviderResponseHandlerTest.java

```diff
@@ -104,9 +105,13 @@ public void testSignIn_autoUpgradeAnonymousEnabledWithExistingUser_expectMergeFa
         mHandler.getOperation().observeForever(mResponseObserver);
         setupAnonymousUpgrade();
 
+        FirebaseAuthUserCollisionException ex =
+                new FirebaseAuthUserCollisionException("foo", "bar");
+        TestHelper.setPrivateField(ex, FirebaseAuthUserCollisionException.class,
```

Comment:
@lsirac you removed the constructor of `FirebaseAuthUserCollision` that we were using, so I have to use a reflection hack to make this test pass.

Can't use a mock because the exception class is `final`

Reply:
I added a #setAuthCredential method, can we just use that? Or will the symbols change on each release?

Reply:
Yeah unfortunately that's proguarded away for 3p:

```java
    public final FirebaseAuthUserCollisionException zza(AuthCredential var1) {
        this.zzhi = var1;
        return this;
    }
```